route_test.go

```diff
@@ -1748,3 +1748,98 @@ func TestRouteUIDOption(t *testing.T) {
 		t.Fatal(routes)
 	}
 }
+
+func TestRouteFWMarkOption(t *testing.T) {
+	tearDown := setUpNetlinkTest(t)
+	defer tearDown()
+
+	// setup eth0 so that network is reachable
+	err := LinkAdd(&Dummy{LinkAttrs{Name: "eth0"}})
+	if err != nil {
+		t.Fatal(err)
+	}
+	link, err := LinkByName("eth0")
```

Comment:
A bit worried about assuming we have `eth0` named interface, but looks like our testing VMs do, UT passed:

![image](https://user-images.githubusercontent.com/10080882/200386441-d18aa0d7-74f9-48be-8ef8-5b2993a809e5.png)